src/client/src/orangehrmPerformancePlugin/pages/EditPerformanceTracker.vue

```diff
@@ -100,15 +99,7 @@ export default {
       required: true,
     },
   },
-  setup() {
-    const http = new APIService(
-      window.appGlobal.baseUrl,
-      '/api/v2/performance/config/trackers',
-    );
-    return {
-      http,
-    };
-  },
+
```

Comment:
why this is removed? you need the APIService to make API calls? I see API calls in this components

Reply:
Added it back